packages/cli/create-strapi-app/create-strapi-app.js

```diff
@@ -51,6 +52,7 @@ async function initProject(projectName, program) {
     return generateApp(projectName, program);
   }
 
+  await checkInstallPath(resolve(projectName));
```

Comment:
Why does this check is done after potentially generating a quickstart app? Shouldn't it be called before the condition above?

Reply:
The check is also done in `generateNewApp` so it will work as expected, but I agree it would be much cleaner. Updating this